storybook/src/Storybook/PagesList.qml

```diff
@@ -21,6 +21,21 @@ ListView {
 
         width: ListView.view.width
 
+        Drag.dragType: Drag.Automatic
+        Drag.active: dragArea.drag.active
+        Drag.supportedActions: Qt.IgnoreAction
+        Drag.mimeData: {
+            "text/uri-list": `file:${pagesFolder}/${model.title}Page.qml`
+        }
+
+        MouseArea {
+            id: dragArea
+            anchors.fill: parent
+
+            drag.target: this
+            acceptedButtons: Qt.RightButton
```

Comment:
Right button? hmmm why :)

Reply:
Yeah, it would be nice to have it under left button as well but then it interferes with dragging the list :/

Reply:
Right, the darned Flickable/Scrollview